src/tokens/TokenBalanceItem.tsx

```diff
@@ -12,14 +12,20 @@ import { TokenBalance } from 'src/tokens/slice'
 
 interface Props {
   token: TokenBalance
+  balanceUsdErrorFallback?: string
   onPress?: () => void
   containerStyle?: ViewStyle
 }
 
-export const TokenBalanceItem = ({ token, onPress, containerStyle }: Props) => {
+export const TokenBalanceItem = ({
+  token,
+  onPress,
+  containerStyle,
+  balanceUsdErrorFallback,
+}: Props) => {
   const { t } = useTranslation()
 
-  const Content = ({ token, containerStyle }: Props) => {
+  const Content = () => {
```

Comment:
Wait, we shoudn't create new components during render. This is bad for perf.
I know it's not directly related to the focus of this PR, we can address separately.

Reply:
no worries, we could just store it in a variable instead of adding another component.

Reply:
why is it preferable to use a variable rather than a normal component?

Reply:
Just for simplicity, I don't see a strong case for having 2 of them.